.github/COMMUNITY_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Community engineering issue
+about: Create an issue related to a community engineering effort
+---
+
+# Community engineering issue
+
+<!-- (REQUIRED) Provide information for the doc request including any Community code issues or PRs, Magento versions, or user guide pages. -->
+
+Type of Doc:
+
+-  [ ] New topic
+-  [ ] Topic Update, user guide link:
+-  [ ] Other
+
+Magento 2 GitHub Issue: https://github.com/magento/merchdocs/issues/link
+
+Magento 2 GitHub PR: https://github.com/magento/merchdocs/pull/link
```

Comment:
it is not clear from the context how the link `https://github.com/magento/merchdocs/pull/link` relates to the text `Magento 2 GitHub PR: `

Reply:
Sam as above, but should probably be the related Community PR. Do you agree?